.github/comment.yml

```diff
@@ -36,11 +62,6 @@
     label: Approved-OkToMerge
     onLabeledComments: "Hi @${PRAuthor},Your PR is approved. Congratulations. </br>  <li> Since your PR is in private repo (azure-rest-api-specs-pr),  there won’t be PR merge. If you want to publish the PR to public repo (Azure/azure-rest-api-specs) and get it merged, pls use [OpenAPIHub Publish PR](${openapiHub}/tools/publishpullrequest?pr=${owner}/${repo}/${PRNumber}&to=${to}).</li> <li> For further guidance on how to proceed.  Please refer to this [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/205/RP-Scenarios-to-Contribute-to-Swagger?anchor=**public-repository-vs.-private-repository**)</li>"
```

Comment:
this is only true if they are targeting main branch in the private repo. Is there a way to configure the comment to only appear in that occasion? If not, t will be good to rephrase it so that becomes more clear

Reply:
Great catch. Fixed.